src/UseCases/ApplyForMembership/MembershipApplicationValidator.php

```diff
@@ -14,8 +14,9 @@
  */
 class MembershipApplicationValidator {
 
-	const MIN_YEARLY_PAYMENT_IN_EURO = 25;
-	const MONTHS_PER_YEAR = 12;
+	/* private */ const MIN_PERSON_YEARLY_PAYMENT_IN_EURO = 24;
+	/* private */ const MIN_COMPANY_YEARLY_PAYMENT_IN_EURO = 100;
+	/* private */ const MONTHS_PER_YEAR = 12;
```

Comment:
I found that I had used a constant I should not have used at some place, and decided to specify my intend properly rather than wait for a new version of the language. When 7.1 comes out, we can go through all the constants and make then private if appropriate in little time.